### PR TITLE
Add layman XML file.

### PR DESCRIPTION
**Pull request by Lucretia**
Please add this.

### CODE REVIEW

File: layman.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+
+<repositories version="1.0">
+	<repo priority="50" quality="experimental" status="unofficial">
+		<name>larry</name>
```
Comment (aballier) [inline layman.xml:5]: shouldn't this match repo_name ?

Comment (Lucretia) [reply layman.xml:5]: yeah, it's a typo

**Comment by XVilka** [comment]
Maybe it makes sense to close this one?

**Comment by aballier** [comment]
superceeded by #2

**Pull request closed without merging**